docs/home/howto/kubernetes/PERSISTENT_STORAGE.md

```diff
@@ -90,6 +90,25 @@ specification:
 
 #### Create disks for Rook/Ceph Cluster Storage - AWS
 
+To define additional empty disk resources for Rook/Ceph Cluster Storage on AWS, use `specification.disks.additional_disks`:
+```yaml
+---
+kind: infrastructure/virtual-machine
+title: Virtual Machine Infra
+provider: aws
+name: kubernetes-node-machine
+specification:
+  size: t3.medium
+  authorized_to_efs: true
+  mount_efs: true
+  disks:
+    additional_disks:
+      - volume_type: gp2
```

Comment:
It will be good to provide information or links what types, size are supported.

Reply:
Added link to aws instance ebs